src/aws_encryption_sdk/identifiers.py

```diff
@@ -240,6 +240,8 @@ def safe_to_cache(self):
         return self.kdf is not KDFSuite.NONE
 
 
+# algorithm is just an alias for AlgorithmSuite ... but Sphinx does not recognize this fact
```

Comment:
I removed the comment and changed the one below to be more concise and readable (so that people would know the import facts: that sphinx linking doesn't work when referencing "Algorithm")